tests/data/wire/ext_conf_no_extensions-block_blob.xml

```diff
@@ -0,0 +1,12 @@
+<Extensions version="1.0.0.0" goalStateIncarnation="9">
+  <GuestAgentExtension xmlns:i="http://www.w3.org/2001/XMLSchema-instance">
+    <GAFamilies>
+    </GAFamilies>
+  </GuestAgentExtension>
+  <Plugins>
+  </Plugins>
+  <PluginSettings>
+  </PluginSettings>
+  <StatusUploadBlob statusBlobType="BlockBlob">http://foo</StatusUploadBlob>
+</Extensions>
+
```

Comment:
The tests for the status blob type were reaching too much into internals and were a pain to maintain each time I change code in the goal state stuff so finally I got tired of them and change them to use the mock protocol. These are the test data files for the mock